multigeiger/multigeiger.ino

```diff
@@ -117,7 +117,7 @@ void display(unsigned long current_ms, unsigned long current_counts, unsigned lo
 
     // ... and display them.
     DisplayGMC(((int)accumulated_time / 1000), (int)(accumulated_Dose_Rate * 1000), (int)(Count_Rate * 60),
-               (showDisplay && switches.display_on), wifi_connected);
+               (showDisplay && switches.display_on), wifi_connected, false); // last false in preparation for future BLE connectivity
```

Comment:
maybe add ble_connected or ble_active as variable to main loop (and put the comment there), and use the variable here?

Reply:
or do not yet add the param / variable at all.

Reply:
I need the variable to add the BLE symbol code. Will add an always-false variable in multigeiger.ino